scripts/dump-remote-schema.js

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable no-console */
```

Comment:
This can be made into a more general-purpose script, to hit any GraphQL endpoint and print out the types.

An alternative might be to generate the type file _from_ the backend service, in Constellation for instance, it's a 1-liner: https://github.com/artsy/constellation/blob/369b1b29f83c4e3b2f91f62e26e46c7506723290/lib/tasks/graphql.rake#L6

But, since there might be different GraphQL backends, and we want to be in control of the schema that's being stitched in, the script should probably live here.

Reply:
Yeah, I agree with having it in here 👍